base_rest/tests/common.py

```diff
@@ -49,56 +50,71 @@ def setUpRegistry(cls):
 
 
 class RestServiceRegistryCase(ComponentRegistryCase):
-    def setUp(self):
-        super().setUp()
 
-        self._service_registry = RestServicesRegistry()
+    # pylint: disable=W8106
+    @staticmethod
+    def _setup_registry(class_or_instance):
+        super().setUpClass()
```

Comment:
setUpClass call here? It should be the other way around: setUpClass calls _setup_registry, no? :thinking:

Reply:
Good catch @simahawk I removed the line since it's useless and is a remaining of the refactoring.